### PR TITLE
Enforce naming convention on apps and roles

### LINKED ISSUE #121: Settle on a role naming convention
Right now we have different variations of naming roles. For example:

- [Create fundraising (action)](https://github.com/aragon/aragon-core/blob/dev/metadata/FundraisingApp.json#L5)
- [Adding member role (present tense of action + "role" suffix)](https://github.com/aragon/aragon-core/blob/dev/metadata/GroupApp.json#L5)
- [Permission creator ("actual" role name)](https://github.com/aragon/aragon-core/blob/dev/metadata/Kernel.json#L5)
- [Issue tokens role (action + "role" suffix)](https://github.com/aragon/aragon-core/blob/dev/metadata/TokenManager.json#L6)

Since this is going to be displayed in the UI, I think it would be nice to settle on a convention.

### LINKED ISSUE #122: Settle on an app naming convention
Right now we have two different naming schemes for apps

- Just the app name
- The app name + "App" suffix

### PR DESCRIPTION
**Pull request by izqui**
App names no longer have the 'App' suffix.

Role names should make this construct grammatically correct 'If granted this role, an entity can [role name]' (e.g: create new payments)

Closes #121 & closes #122

### CODE REVIEW

File: metadata/Kernel.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
     "appName": "kernel.aragonpm.test",
     "version": "1.0.0",
     "roles": [
-        { "name": "Permission creator", "id": "PERMISSION_CREATOR_ROLE" },
-        { "name": "App upgrader", "id": "APP_UPGRADER_ROLE" },
-        { "name": "Kernel upgrader", "id": "KERNEL_UPGRADER_ROLE" }
+        { "name": "Create new permissions", "id": "CREATE_PERMISSIONS_ROLE" },
+        { "name": "Upgrade apps", "id": "UPGRADE_APPS_ROLE" },
+        { "name": "Kernel upgrader", "id": "UPGRADE_KERNEL_ROLE" }
```
Comment (onbjerg) [inline metadata/Kernel.json]: "Upgrade the kernel"

File: metadata/Voting.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "appName": "voting.aragonpm.test",
+    "version": "1.0.0",
+    "roles": [
+        { "name": "Votes creator", "id": "CREATE_VOTES_ROLE" },
```
Comment (onbjerg) [inline metadata/Voting.json]: "Create votes"

File: metadata/Voting.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+        { "name": "Quorum modifier", "id": "MODIFY_QUORUM_ROLE" }
```
Comment (onbjerg) [inline metadata/Voting.json]: "Modify quorum"

**Comment by izqui** [comment]
Ups, good catch!

**Pull request merged**